src/go/tokengenerator/token_generator.go

```diff
@@ -106,7 +106,7 @@ func generateAccessToken(keyData []byte) (string, time.Duration, error) {
 func MakeLatsHandler(serviceAccountKey string) http.Handler {
 	r := mux.NewRouter()
 
-	r.PathPrefix(util.AccessTokenPath).Methods("GET").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	r.PathPrefix(util.LatsAccessTokenPath).Methods("GET").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can we have consistent naming? for port, we call it TokenAgent,  or path we call it LatsAccessToken.   Can we all call it TokenAgent